### PR TITLE
[W5.4][T15-A3]Cheng Jin Ting

### PR DESCRIPTION
**Pull request by jin-ting**
Pull request for week5 LO

### CODE REVIEW

File: src/seedu/addressbook/commands/FindCommand.java [hunk 1]
```diff
@@ -24,7 +24,7 @@
     private final Set<String> keywords;
 
     public FindCommand(Set<String> keywords) {
-        this.keywords = keywords;
+        this.keywords = keywords ;
```
Comment (okkhoy) [inline src/seedu/addressbook/commands/FindCommand.java]: err... no need to add space between word and semicolon.

File: test/java/seedu/addressbook/commands/FindCommandTest.java [hunk 2]
```diff
@@ -26,7 +26,7 @@ public void execute() throws IllegalValueException {
         assertFindCommandBehavior(new String[]{"Amy"}, Arrays.asList(td.amy));
 
         //same word, different case: not matched
-        assertFindCommandBehavior(new String[]{"aMy"}, Collections.emptyList());
+        assertFindCommandBehavior(new String[]{"aMy"},Collections.emptyList());
```
Comment (okkhoy) [inline test/java/seedu/addressbook/commands/FindCommandTest.java]: where is the test for the `sort` command you implemented?

**Comment by okkhoy** [comment]
@jin-ting 
Some comments added. Please ack & close the PR after reading comments.

**Comment by jin-ting** [comment]
Updated Junit for SortCommand.
Closing PR

**Pull request closed without merging**